nautobot/core/settings.json

```diff
@@ -0,0 +1,544 @@
+{
+    "type": "object",
+    "properties": {
+        "ADMINS": {
+            "type": "array",
```

Comment:
Sorry if I was unclear in the previous comments - IMO every `type: array` in this schema should have a corresponding `items: { type: ... }` to specify what the array contains.